### ISSUE TITLE
Can't find the Wifi AP network!

### ISSUE THREAD

**Issue by BullzLabz** [HomeKidd/Homekit-WS2812B-controller#51]
Hello,

Sorry to open a new issue for this problem (again) but I can't find a good solution to fix it!

I used this ([Dev Board](https://a.aliexpress.com/_mLadC5z) for flash the "ledstrip.bin" file to my ESP12-F and I try several times with 3 different ESP-12F modules and I can't see the Wifi AP network !

I've also tried different flash mode "DIO" and "QIO" but same issue and I can't choose the flash size with the NodeMCUFlash tool!

What's doing wrong? I missing steps?

This is the result of my flash's :

```
esptool.py v2.6
Serial port COM5
Connecting....
Detecting chip type... ESP8266
Chip is ESP8266EX
Features: WiFi
MAC: c4:5b:be:6e:24:f5
Uploading stub...
Running stub...
Stub running...
Configuring flash size...
Auto-detected Flash size: 4MB
Erasing flash (this may take a while)...
Chip erase completed successfully in 3.4s
Compressed 507328 bytes to 358349...
Wrote 507328 bytes (358349 compressed) at 0x00000000 in 32.4 seconds (effective 125.5 kbit/s)...
Hash of data verified.

Leaving...
Staying in bootloader.

Firmware successfully flashed. Unplug/replug or reset device 
to switch back to normal boot mode.
```

**Comment by BullzLabz** [comment]
Hello,

One more day has tried to find the Wifi network **"HomeKid-xxxx"** by flashing other ESP-12F modules and still nothing.

For additional info, I even tried to flash an ESP-32 (Wemos Wroom 32) and nothing either, still no Wifi network with the name of **HomeKid-xxxx**

I tested a blank module (ESP-12F) to see if I found its network, and no problem with the modules, they all seem to work fine, their WiFi networks are displayed and I can connect it.

For additional information, I'm on Windows 10 and the only thing that I didn't tried is the software provided by **Espressif**, because I may have made a mistake, it's a real gas factory!

Your help would be really appreciated because I received my printed circuit order via PCBWay and I only have this problem to solve to go to the next step, the reflow!
